template_sample-wds-table.tsv

```diff
@@ -0,0 +1,10 @@
+sys_name	column-1-file-type-i-e-CRAM	column-2-file-type-i-e-collection-date
```

Comment:
Does this file need to be checked in? If it does, could we put it somewhere other than the repository root? Maybe a `sample-data` folder or something like that?

Reply:
Ah yeah I threw it in root just because I wasn't expecting to keep it in the repository, just there until we find a better home for it.  I'll move it someplace better at least for now; potentially end up removing it all together.